conf/ams-create-users-gocdb.template

```diff
@@ -0,0 +1,28 @@
+[AMS]
+# under which ams project, the users will be created
+ams_project:
+# goc db url to pull user data
+goc_db_host:
+# service types referes to the different service types that will we should keep from the xml and assign them to the respectivew ams topic
+service-types:
+# ams use role
```

Comment:
Add options where possible eg. Publisher, Subscriber
OIDC, x509 etc so that its each easy to copy paste

Reply:
ready